fixtures/robe/colorspot-2500e-at.json

```diff
@@ -845,139 +896,125 @@
       "capabilities": [
         {
           "dmxRange": [0, 9],
-          "type": "GoboIndex",
-          "index": 0,
-          "comment": "open",
+          "type": "WheelSlot",
+          "slotNumber": 1,
           "switchChannels": {
             "Gobo 1 indexing/rotation": "Gobo 1 indexing",
             "Gobo 1 indexing/rotation fine": "Gobo 1 indexing fine"
           }
         },
         {
           "dmxRange": [10, 13],
-          "type": "GoboIndex",
-          "index": 1,
-          "comment": "Rainbow Chips 15030012 (indexed)",
+          "type": "WheelSlot",
+          "slotNumber": 2,
```

Comment:
TODO: add `"comment": "indexed"` / `"comment": "rotation"` to all these capabilities